### ISSUE TITLE
Cannot open grind manager view

### ISSUE THREAD

**Issue by eduongAZ** [dromse/obsidian-grind-manager#4]
System: Mac, tested on fresh vault.

Problem: When I click on the "show grind manager" button, it opens a blank view. I inspected the console and I get the following error:

```
app.js:1 Failed to open view ReferenceError: React is not defined
    at ri.onOpen (plugin:grind-manager:15:14577)
    at ri.<anonymous> (app.js:1:1432284)
    at app.js:1:237056
    at Object.next (app.js:1:237161)
    at app.js:1:236077
    at new Promise (<anonymous>)
    at v (app.js:1:235822)
    at t.open (app.js:1:1432141)
    at t.<anonymous> (app.js:1:2041022)
    at app.js:1:237056
```

**Comment by dromse** [comment]
Hi! Thank you for bringing this to attention. I checked this on Linux and this error the same. I'll try to fix it today or the next few days.

**Comment by dromse** [comment]
@eduongAZ I found why it works for me, it's a little hack but before I fix it, you can use it: install excalidraw plugin, it installs react dep and everything would work.

**Comment by dromse** [comment]
@eduongAZ I fixed this issue with 475e849c7c6b32f689ed0ceaa5bf8ad37854b585, try to update grind manager to version `1.0.3`.
